app/src/main/java/org/jellyfin/androidtv/data/eventhandling/TvApiEventListener.java

```diff
@@ -74,11 +75,15 @@ public void onLibraryChanged(ApiClient client, LibraryUpdateInfo info) {
     @Override
     public void onPlaystateCommand(ApiClient client, PlaystateRequest command) {
         PlaybackController playbackController = TvApp.getApplication().getPlaybackController();
+        Timber.d("caught playstate command: %s", command.getCommand());
+
+        //handler to access the players on the main thread
+        Handler mainThreadHandler = new Handler(Looper.getMainLooper());
```

Comment:
It's probably more efficient to create one handler as class property and re-use that one

Reply:
done